packages/lib/CalendarService.ts

```diff
@@ -189,21 +189,46 @@ export default abstract class BaseCalendarService implements Calendar {
       }
 
       const eventsToUpdate = events.filter((e) => e.uid === uid);
+      let ev = eventsToUpdate[0];
       return Promise.all(
         eventsToUpdate.map((e) => {
+          ev = e;
```

Comment:
```suggestion
        eventsToUpdate.map(({eventItem) => {
          event = eventItem;
```
Here we change e for -> eventItem as some people can use e as error sometimes. With this change we add more readability.